### ISSUE TITLE
this pluggin has issue

### ISSUE THREAD

**Issue by amitkumarjain2** [ravid7000/table-sortable#15]
This plugin has following flows.

1. It's working in IE Edge not in IE
2. Header format is still not working, column headers disappearing 
3. We cannot fix the width of each column, if there is more data whole table disturb when move to page no 1 to 2 or so on.
4. Unable change alternate rows color

**Comment by ravid7000** [comment]
1. Which version of IE you are testing?
2. Which version of plugin you are using?
3 and 4 can be fixed using css styling as of now.

**Comment by amitkumarjain2** [comment]
1. IE 11
2. Latest version which is mention in this blog.

How to make alternate row color

**Comment by amitkumarjain2** [comment]
https://github.com/ravid7000/table-sortable/tree/master/dist

I using the index.html file from this folder.

**Comment by ravid7000** [comment]
Use these styles for stripped row

```css
.gs-table tbody tr:nth-of-type(odd) {
  background-color: rgba(0,0,0,.05);
}
```

**Comment by amitkumarjain2** [comment]
Hi Ravi,

Have you identified why this plug in not working in IE11 version?

**Comment by ravid7000** [comment]
Documentation fixed for IE11. It should work now.
https://github.com/ravid7000/table-sortable/pull/17

**Comment by amitkumarjain2** [comment]
Yes, its not working but stopped paging, only "prev" and "next" button in paging is working but when we click on  page number like 1, 2, 3 it's not working in IE11
